### PR TITLE
Fixed some HTML tags.

### PR DESCRIPTION
**Pull request by javadft**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ function prettyDeltaTime($reference)
 <?endif;?>
 <td><?=prettyDeltaTime($row['reap']);?></td>
 <td><?=prettyDeltaTime($row['time']);?></td>
-</td>
+</tr>
```
Comment (jhon) [inline index.php:77]: Haha whoops. How has this been working?

File: index.php [hunk 2]
```diff
@@ -83,9 +83,6 @@ function prettyDeltaTime($reference)
 <td><?=$totalStay?></td>
 <td><?=$totalAbandon?></td>
 <td><?=$totalAbstain?></td>
-<td/>
-<td/>
-<td/>
```
Comment (jhon) [inline index.php:88]: These are intentionally there. They're the empty timestamp columns.

**Comment by javadft** [comment]
Firefox shows them as errors. Plus, shouldn't they be `</td>`?
Good job on the site by the way.

**Comment by jhon** [comment]
Probably. I'm not a web guy :X Change 'em to <td></td> and I'll pull this in

**Comment by jhon** [comment]
Merged this in and readded the empty columns :)

**Pull request merged**